java/client/src/main/java/glide/api/commands/HyperLogLogBaseCommands.java

```diff
@@ -0,0 +1,51 @@
+/** Copyright GLIDE-for-Redis Project Contributors - SPDX Identifier: Apache-2.0 */
+package glide.api.commands;
+
+import java.util.concurrent.CompletableFuture;
+
+/**
+ * Supports commands and transactions for the "HyperLogLog Commands" group for standalone clients
+ * and cluster clients.
+ *
+ * @see <a href="https://redis.io/commands/?group=hyperloglog">HyperLogLog Commands</a>
+ */
+public interface HyperLogLogBaseCommands {
+
+    /**
+     * Adds all the elements to the HyperLogLog data structure stored and create a new structure if it
+     * is missing.
+     *
+     * <p>As a side effect of this command the HyperLogLog internals may be updated to reflect a
```

Comment:
Documentation should be much more concise I believe this is too much. 1-3 sentences is the maximum I have seen in the Node and PY clients.